packages/amplify-util-uibuilder/package.json

```diff
@@ -13,12 +13,17 @@
     "access": "public"
   },
   "dependencies": {
-    "@aws-amplify/codegen-ui": "1.2.0",
-    "@aws-amplify/codegen-ui-react": "1.2.0",
+    "@aws-amplify/codegen-ui-old": "npm:@aws-amplify/codegen-ui@1.2.0",
+    "@aws-amplify/codegen-ui-react-old": "npm:@aws-amplify/codegen-ui-react@1.2.0",
+    "@aws-amplify/codegen-ui-new": "npm:@aws-amplify/codegen-ui@2.1.0",
```

Comment:
What's the reason for using 2 different versions?

Reply:
the premise of this PR is that, since the Studio API schema is changing, we need to detect which type of schema was returned and pass it to the version of codegen-ui that is capable of handling that structure.

This is temporary, as soon everyone will be using the new stuff and we'll be able to use only the new version, but there's a sort of chicken and egg problem where we need to have a backwards compatible cli first before we can migrate the schema.